articles/service-fabric/how-to-managed-cluster-availability-zones.md

```diff
@@ -218,6 +218,26 @@ Requirements:
    ```
    If you run in to any problems reach out to support for assistance.
 
+## Enable FastZonalUpdate on Service Fabric managed clusters (preview)
+Service Fabric managed clusters support faster cluster and application upgrades by reducing the max upgrade domains per availability zone from five to three.  
+
+The update should be done via ARM template by setting the fastZonalUpdate property to “fast”.  The Service Fabric managed cluster resource apiVersion should be 2023-01-01-preview or later
+```json
+   "resources": [
+        {
+            "type": "Microsoft.ServiceFabric/managedClusters",
+            "apiVersion": "2023-01-01-preview",
+            '''
+            "properties": {
+                '''
+                "zonalResiliency": true,
+                "zonalUpdateMode": “fast”,
+                ...
+            }
+        }]
+```
+If you run in to any problems reach out to support for assistance.
```

Comment:
We should also talk about the need to update each nodetype after updating the cluster resource, to fix the UDs on existing nodetypes.

Reply:
Added

Reply:
Please update this Abhay. We need to explicitly say that each nodetype needs to be updated for the new UD scheme to become effective. Then describe how to do it via the 2 steps you listed above in 2 & 3.

Reply:
done